api/src/main/java/com/redhat/ipaas/api/v1/model/ConnectorGroup.java

```diff
@@ -21,30 +21,30 @@
 import java.io.Serializable;
 
 /**
- * ComponentGroups are labels in Camel.
+ * ConnectorGroups are labels in Camel.
```

Comment:
@jimmidyson  @rhuss The userDefined Labels are called "Tags", ConnectorGroups are driven by Camel Labels to Group all IPaasConnectors for let's say "Twitter". Let's not get hung up on this until we fully flesh out how Actions are going to work.